core/src/cost_update_service.rs

```diff
@@ -119,8 +119,10 @@ impl CostUpdateService {
     ) -> u64 {
         let mut update_count = 0_u64;
         for (program_id, program_timings) in &mut execute_timings.details.per_program_timings {
-            let current_estimated_program_cost =
-                cost_model.read().unwrap().find_instruction_cost(program_id);
+            let current_estimated_program_cost = cost_model
+                .read()
+                .unwrap()
+                .find_average_program_cost(program_id);
```

Comment:
Is this fix the the only functional change as compared to the reverted EMA change?